src/Components/WebAssembly/WebAssembly.Authentication/src/Services/RemoteAuthenticationService.cs

```diff
@@ -42,6 +42,11 @@ public class RemoteAuthenticationService<TRemoteAuthenticationState, TProviderOp
         /// </summary>
         protected readonly NavigationManager _navigation;
 
+        /// <summary>
+        /// The <see cref="UserFactory{TAccount}"/> to map accounts to <see cref="ClaimsPrincipal"/>.
+        /// </summary>
+        protected readonly UserFactory<TAccount> _userFactory;
```

Comment:
Should this be a property? It's really odd to see underscores in the ref API

Reply:
Looks like it should be `private`. I don't see any subclasses consuming it (and if they did, then it should definitely be a property).

Reply:
Made all of them a Property. I don't want subclasses having to define their own backing storage, given that they are already forced to inject them on the constructor. As a result the best answer is to make them properties.